cookbooks/aws-parallelcluster-config/recipes/nfs.rb

```diff
@@ -22,10 +22,6 @@
 # NFS threads enhancement is omitted for Ubuntu1604
 node.force_override['nfs']['threads'] = [node['cpu']['cores'].to_i, 8].max
 
-if platform?('ubuntu') && node['platform_version'].to_f >= 16.04
-  # FIXME: https://github.com/atomic-penguin/cookbook-nfs/issues/93
-  include_recipe "nfs::server"
-end
 include_recipe "nfs::server4"
```

Comment:
is this needed? Can this be moved in place of https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-install/recipes/base.rb#L125 ?

Reply:
if we are calling `https://github.com/aws/aws-parallelcluster-cookbook/pull/1528` just to restart the service after the thread configuration, I'm for doing it "by hand" restarting the service, because calling the whole recipe may perform additional (not needed) actions

Reply:
The service restart seems to have been necessary for [Ubuntu1604](https://github.com/aws/aws-parallelcluster-cookbook/blob/12f9ecaae38afd9c4b505dccb15e3d10732bc4f4/cookbooks/aws-parallelcluster-config/recipes/nfs.rb#L18) to pick up the changes. It may no longer be needed in 18_04 or 20_04. I'll check on this.

Reply:
> The service restart seems to have been necessary ...

I think it's still needed for all the OS after setting 
`node.force_override['nfs']['threads'] = [node['cpu']['cores'].to_i, 8].max`
so that the service see the config change